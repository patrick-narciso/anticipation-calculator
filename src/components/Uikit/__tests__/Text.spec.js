// eslint-disable-next-line no-unused-vars
import { dom } from 'isomorphic-jsx';
import { Text } from '@Uikit';

test('Should be render Text Component correctly', () => {
  expect(<Text />).toMatchSnapshot();
});

test('Should be render Text Component Bold variant correctly', () => {
  expect(<Text bold />).toMatchSnapshot();
});

test('Should be render Text Component Detail variant correctly', () => {
  expect(<Text detail />).toMatchSnapshot();
});

test('Should be render Text Component Medium variant correctly', () => {
  expect(<Text medium />).toMatchSnapshot();
});

test('Should be render Text Component Reduced variant correctly', () => {
  expect(<Text reduced />).toMatchSnapshot();
});
