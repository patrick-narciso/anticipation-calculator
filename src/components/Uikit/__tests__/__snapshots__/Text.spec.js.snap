// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should be render Text Component Bold variant correctly 1`] = `"<span class=\\"css-uw8vfg-TextBold-text\\"></span>"`;

exports[`Should be render Text Component Detail variant correctly 1`] = `"<span class=\\"css-t1t0y7-TextDetail-text\\"></span>"`;

exports[`Should be render Text Component Medium variant correctly 1`] = `"<span class=\\"css-1sgkqby-TextMedium-text\\"></span>"`;

exports[`Should be render Text Component Reduced variant correctly 1`] = `"<span class=\\"css-18alwqt-TextReduced-text\\"></span>"`;

exports[`Should be render Text Component correctly 1`] = `"<span class=\\"css-1rtppnv-text\\"></span>"`;
